app/views/home/index.html.slim

```diff
@@ -35,7 +35,7 @@
       p.font-xsmall
         | Search for an existing measure on the database to view or make changes
 
-  - if Rails.env.development?
+  - if Rails.env.development? || ENV['GOVUK_APP_DOMAIN'] == 'tariffs-dev'
```

Comment:
we could encapsulate that into a module variable like we do on TT backend

Reply:
@matthewford agree :+1: will do